docs/cinder-csi-plugin/using-cinder-csi-plugin.md

```diff
@@ -96,7 +96,6 @@ Implementation of `cinder-csi-plugin` relies on following OpenStack services.
 
 | Service                        | API Version(s) | Deprecated | Required |
 |--------------------------------|----------------|------------|----------|
```

Comment:
just curious we  are talking about block storage version? why remove keystone version here?

Reply:
@jichenjc its also outdated , so removed it. Ill update the description . Thanks